addons/survey/wizard/survey_invite.py

```diff
@@ -43,7 +43,12 @@ def _get_default_author(self):
         help="Author of the message.")
     # recipients
     partner_ids = fields.Many2many(
-        'res.partner', 'survey_invite_partner_ids', 'invite_id', 'partner_id', string='Recipients')
+        'res.partner', 'survey_invite_partner_ids', 'invite_id', 'partner_id', string='Recipients',
+        domain="""[
+            '|', (survey_users_can_signup or not survey_users_login_required, '=', 1),
```

Comment:
'|', '|',
(survey_users_can_signup, '=', 1)
(not survey_users_login_required, '=', 1)
('user_ids', '!=', False)

Reply:
Works for me.